.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+# Welcome to Cryostat! 👋
+## Before contributing, make sure you have:
+* [ ] Read the [contributing guidelines](https://github.com/cryostatio/cryostat/blob/main/CONTRIBUTING.md)
```

Comment:
should point to the CONTRIBUTING.md file in cryostat3

Reply:
yes

Reply:
also while you're at it, maybe you could link the contributing doc in README?